var.tfvars

```diff
@@ -0,0 +1,5 @@
+location  = "uk south"
```

Comment:
tfvars should not be in the module, if you are using it for testing then create a sub-folder called `example`, see https://github.com/hmcts/cnp-module-postgres/tree/master/example

Reply:
I had .tfvars in .gitignore already. Which should have been ignore.  Will verify the file once again.